common/static/common/js/discussion/views/discussion_content_view.js

```diff
@@ -415,17 +415,19 @@
                 updates = {
                     upvoted_ids: (is_voting ? _.union : _.difference)(user.get('upvoted_ids'), [this.model.id])
                 };
-                return DiscussionUtil.updateWithUndo(user, updates, {
-                    url: url,
-                    type: "POST",
-                    $elem: $(event.currentTarget)
-                }, gettext("We had some trouble saving your vote.  Please try again.")).done(function() {
-                    if (is_voting) {
-                        return self.model.vote();
-                    } else {
-                        return self.model.unvote();
-                    }
-                });
+                if (!$(event.target.closest(".actions-item")).hasClass('is-disabled')) {
```

Comment:
For the record: Brian and I discussed how it would be better for the Underscore template not to output the link, but it isn't provided with enough context, and it would be too difficult to wire the context through. So for now we agreed that adding this logic is the way to go.